### DISCUSSION TITLE
Welcome to poketwo-Autocatcher Discussions!

### DISCUSSION THREAD

**Discussion by HOPE-NEXUS** [Team-BANERUS/poketwo-Autocatcher#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lee-hunter1** [comment]
Hey i was wondering the discord link is expired if I could add you for the bot?
